tests/issues/GH4313.test.ts

```diff
@@ -0,0 +1,121 @@
+import { Entity, MikroORM, PrimaryKey, Property, UnderscoreNamingStrategy } from '@mikro-orm/core';
+import { MongoDriver } from '@mikro-orm/mongodb';
+import { closeReplSets, initMongoReplSet } from '../bootstrap';
+import { randomUUID } from 'crypto';
+
+@Entity()
+export class A {
+
+  @PrimaryKey()
+  _id = randomUUID();
+
+  @Property()
+  complexName!: string;
+
+}
+
+let orm: MikroORM<MongoDriver>;
+
+function testCase() {
+  test('Insert entities', async () => {
+    const a = new A();
+    a.complexName = 'a';
+    const b = new A();
+    b.complexName = 'b';
+    await orm.em.persistAndFlush(a);
+    await orm.em.persistAndFlush(b);
```

Comment:
you can pass both in a single call
```suggestion
    await orm.em.persistAndFlush([a, b]);
```